task/backend/scheduler/treescheduler.go

```diff
@@ -255,14 +255,12 @@ func (s *TreeScheduler) iterator(ts time.Time) (btree.ItemIterator, *itemList) {
 					s.onErr(context.Background(), it.id, it.Next(), &ErrUnrecoverable{err})
 					return true
 				}
+				it.incrementNonce()
```

Comment:
As we discussed, please remove all references to `nonce` from the Item struct, `Less` function and here, as it provides no value today and raises more questions for other engineers examining the code.

Reply:
ok, made the change.